internal/pkg/cli/task_run.go

```diff
@@ -115,8 +120,7 @@ func newTaskRunOpts(vars runTaskVars) (*runTaskOpts, error) {
 		if err := opts.configureRepository(provider); err != nil {
 			return err
 		}
-		// TODO: configure runner
-		return nil
+		return opts.configureRunner(provider)
```

Comment:
I wonder if we should the session initialization in the `configureRuntimeOpts` func so that the same session can be passed down to both `configureRepository` and `configureRunner`. Because `task.EnvRunner` should initalize its dependencies using the EnvManagerRole as well.

Reply:
What do you think of this:
```go
sess := // blah blah initialize appropriate session
o.repository, err := opts.newRepository(sess)
if err != nil { return err }
o.runner, err := opts.newRunner(sess)
if err != nil { return err }
```